python/py-kaitaistruct/Portfile

```diff
@@ -4,25 +4,23 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-kaitaistruct
-version             0.8
+version             0.9
 platforms           darwin
 license             MIT
 maintainers         nomaintainer
 
 description         Kaitai Struct declarative parser generator for binary data runtime \
                     library for Python
-long_description    ${description}
+long_description    {*}/${description}
 
 homepage            http://kaitai.io
-master_sites        pypi:k/kaitaistruct
-distname            kaitaistruct-${version}
 
-checksums           md5     dadc1ff2bd3958246214c0f546306bc7 \
-                    rmd160  bc96d8d4e051319b4e5b895b53dbc8702f90fd25 \
-                    sha256  d1d17c7f6839b3d28fc22b21295f787974786c2201e8788975e72e2a1d109ff5 \
-                    size    5158
+checksums           md5     2dc3c4226d1188421959069ad73c793b \
```

Comment:
`md5` checksum can be removed, keeping only `rmd160`, `sha256`, and `size`.

Reply:
Thanks done.

I also removed the last unnecessary `livecheck` setting, and snuck in whitespace corrections that aren't going to be fixed otherwise.